lib/formatters/html.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @fileoverview HTML reporter
+ * @author Julian Laval
+ * @copyright 2015 Julian Laval. All rights reserved.
+ */
+"use strict";
+
+var Handlebars = require("handlebars");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Given a word and a count, append an s if count is not one.
+ * @param {string} word A word in its singular form.
+ * @param {int} count A number controlling whether word should be pluralized.
+ * @returns {string} The original word with an s on the end if count is not one.
+ */
+function pluralize(word, count) {
+    return (count === 1 ? word : word + "s");
+}
+
+/**
+ * Renders text along the template of x problems (x errors, x warnings)
+ * @param {string} totalErrors Total errors
+ * @param {string} totalWarnings Total warnings
+ * @returns {string} The formatted string, pluralized where necessary
+ */
+Handlebars.registerHelper("renderText", function(totalErrors, totalWarnings) {
```

Comment:
We should probably create a new instance of handlebars so as not to pollute the global reference. 

`var handlebars = Handlebars.create()`